libandroid-navigation-ui/.gitignore

```diff
@@ -1 +1,2 @@
-dependency-graph-mapbox-libraries.png
\ No newline at end of file
+dependency-graph-mapbox-libraries.png
+com.mapbox.services.android.navigation.ui.v5
```

Comment:
The name of this file is same as application id which happens to be the package name by default most of the times. So this kind of simple pattern matching might cause more files or contents ignored as false positives. I would suggest removing this here and recommend adding **sdk_versions/** to the project level .gitignore file, so that this folder and its contents would be ignored anywhere in the project.

Reply:
Yeah I opted to do this because you can't commit empty folders in git. There's a hacky workaround though https://stackoverflow.com/questions/4250063/how-to-gitignore-all-files-folder-in-a-folder-but-not-the-folder-itself Will implement.

Reply:
Implemented in https://github.com/mapbox/mapbox-navigation-android/pull/2666/commits/16231999cb66606dcfe993112aaac68f30005a70 @harvsu